Source/WebCore/Modules/cookie-store/CookieStore.cpp

```diff
@@ -98,7 +98,7 @@ void CookieStore::MainThreadBridge::ensureOnMainThread(Function<void(ScriptExecu
 {
     ASSERT(m_cookieStore);
 
-    RefPtr context = m_cookieStore->scriptExecutionContext();
+    RefPtr context = RefPtr { m_cookieStore.get() }->scriptExecutionContext();
```

Comment:
I think the preferred convention is to add protectedCookieStore which returns a Ref and call that here.